cmd/osquery-perf/mac10.14.6.tmpl

```diff
@@ -351,8 +351,14 @@
     "version": "{{ .Version }}",
     "type": "Application (macOS)",
     "bundle_identifier": "{{ .BundleIdentifier }}",
-    "source": "apps"
+    "source": "apps",
+    {{/* Note that in Go < 1.18, `{{ or (and .LastOpendedAt .LastOpenedAt.Unix) "" }}` won't work as expected because "and" and "or" don't short circuit. This was changed in Go 1.18 */}}
+    {{if .LastOpenedAt}}
+    "last_opened_at": "{{ .LastOpenedAt.Unix }}"
+    {{else}}
+    "last_opened_at": ""
```

Comment:
Nit, looks like it returns -1 when there's no last opened at?
https://github.com/osquery/osquery/blob/d8537abe36e81617ea15cf468192ef1612760d24/osquery/tables/system/darwin/apps.mm#L191-L203

Reply:
Yeah, good point. This is properly handled in fleet either way, but I'll change it in a future PR.